saos-webapp/src/main/resources/message/common_pl.properties

```diff
@@ -130,6 +130,11 @@ judgmentSearch.results.paginationInfo = Pozycje od {0} do {1} spośród {2} wyni
 judgmentSearch.results.judgmentsNumber = {0,choice,0#0 orzeczeń|0<{0} orzeczenie|1<{0} orzeczenia|4<{0} orzeczeń} 
 judgmentSearch.results.link = Przejdź do strony szczegółów orzeczenia 
 
+judgmentSearch.courtType.COMMON = Sądu Powszechnego
+judgmentSearch.courtType.SUPREME = Sądu Najwyższego
+judgmentSearch.courtType.ADMINISTRATIVE = Sądu Administracyjnego
+judgmentSearch.courtType.CONSTITUTIONAL_TRIBUNAL = Trybunału Konstytucyjnego
+judgmentSearch.courtType.NATIONAL_APPEAL_CHAMBER = Krajowej Izby Odwoławczej
```

Comment:
Could you add 'results', I mean judgmentSearch.results.courtType..., in order for it to be more clear that these properties relate to the result part of the page.